examples/eks-cluster-with-observability/README.md

```diff
@@ -0,0 +1,146 @@
+# EKS Cluster with Observability Tools
+
+This example deploy a new Kubernetes Cluster with Amazon Managed Prometheus, Amazon Managed Grafana, FluentBit, and Opensearch. It also includes instructions for deploying a sample workload with ArgoCD to generate logs and metrics.
+
+---
+**NOTE**
+
+For the sake of simplicity in this example, we store sensitive information and credentials in `dev.tfvars`. This should not be done in a production environment. Instead, store the information in AWS Secrets Manager.
```

Comment:
and use [aws_secretsmanager_secret](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/secretsmanager_secret) to retrieve it.

Reply:
Thanks! Updated.